BuildResidentialHPXML/measure.rb

```diff
@@ -60,6 +60,15 @@ def arguments(model)
     arg.setDescription('Absolute/relative path of the HPXML file.')
     args << arg
 
+    occupancy_calculation_type_choices = OpenStudio::StringVector.new
+    occupancy_calculation_type_choices << 'asset'
+    occupancy_calculation_type_choices << 'operational'
+
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('occupancy_calculation_type', occupancy_calculation_type_choices, true)
+    arg.setDisplayName('Occupancy Calculation Type')
+    arg.setDescription("The type of occupancy calculation type. If 'asset' is chosen, usage multipliers not provided continue to default to 1.0 (meaning energy use is calculated based on floor area, # bedrooms, etc.). If 'operational' is chosen, usage multipliers not provided are calculated based on # occupants, floor area, # bedrooms, etc.")
+    args << arg
```

Comment:
Did you test this? Looking at the code suggests this will have no impact because all usage multiplier arguments throughout this measure are required, are of type `double`, and default to 1.0. So for an operational calculation, how would someone let the default usage multipliers apply?

Reply:
I wouldn't pay attention to this argument description. (That's what you're referring too, right?) This was our original approach, but has since changed. The approach now is to adjust usage multipliers no matter their value (default value or not).

Reply:
Oh, right! Thanks for reminding me, I had forgotten which approach we ended up with. I'll review with that in mind.